.github/workflows/deploy.yml

```diff
@@ -76,7 +77,7 @@ jobs:
           "
       # Configure
       - name: Enable plugins
-        run: $SSH "$TUTOR plugins enable mfe indigo android forum"
+        run: $SSH "$TUTOR plugins enable mfe indigo android forum codejail"
```

Comment:
Don't we have to run `tutor local init --limit=codejail` prior to `local launch`, as per the codejail readme? https://github.com/eduNEXT/tutor-contrib-codejail/#usage

Reply:
After I manually installed and enabled the plugin I can confirm that init is required.

Reply:
Done :)